config/config.js

```diff
@@ -4,4 +4,5 @@ var _ = require('underscore');
 
 module.exports = _.extend(
     require(__dirname + '/../config/env/all.js'),
-    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.js') || {});
\ No newline at end of file
+    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.js') || {});
+    
```

Comment:
hey, one escaped you.

Reply:
Line 8 is empty. It can serve as a new line right ?. Is there something I'm missing ?.